tests/e2e/fixtures/statsFixtures.js

```diff
@@ -326,6 +326,90 @@ const seasonThreeMatches = [
   },
 ];
 
+const gameFixtures = [
```

Comment:
Should probably add some additional games to test some edge cases:
1) game in a different season
2) game before first season
3) game after last season

Reply:
Speaking of  testing, and I don't recall if this is set up, but I think `getSeasonFromTimeStamp` is ripe for some unit tests just because there are so many different cases:

* no seasons
* 2 seasons (which is tested here)
* 3 seasons
* game before the first season
* game after the last season

Most of those cases probably aren't possible, but having the backend error out for such an avoidable reason feels silly.  Setting up all of these in `gameFixtures` is possible, but I imagine can get annoying and the tests end up kinda removed from the thing that they're testing

Reply:
I also updated the cypress test so that it loads a fixture that includes:
1. A game before the first season
2. A game after the last season
3. Games in each season across multiple weeks

And asserts that the endpoint responds with the expected data. This isn't as thorough as the unit testing we can get in #475 , but it does address the most essential cases imo